docker-compose.yml

```diff
@@ -269,9 +269,6 @@ services:
       BAYESIAN_GREMLIN_HTTP_SERVICE_PORT: "8182"
       BAYESIAN_GREMLIN_HTTPINGESTION_SERVICE_HOST: "bayesian-gremlin-http"
       BAYESIAN_GREMLIN_HTTPINGESTION_SERVICE_PORT: "8182"
-      AWS_S3_IS_LOCAL: 1
-      AWS_S3_ACCESS_KEY_ID: "GNV3SAHAHA3DOT99GQII"
-      AWS_S3_SECRET_ACCESS_KEY: "ZmvMwngonaDK5ymlCd6ptaalDdJsCn3aSSxASPaZ"
```

Comment:
@msrb, wouldn't we need these credentials. The credentials here wouldn't work, but we do need the correct credentials. Is it right to remove these 3 lines?

Reply:
This is the right thing to do if we don't want to spend a lot of time refactoring data-importer (which we will need to do in future). data-importer decides whether it is being deployed locally based on whether those AWS credentials are set or not. It uses different env. variables for configuring Minio credentials, and values for those are hardcoded in source code.

It is complicated and that is what makes it error-prone and hard to maintain.